src/main/java/com/rc/porms/controllers/search/SearchOffenseController.java

```diff
@@ -3,7 +3,7 @@
 import com.rc.porms.appl.model.offense.Offense;
 import com.rc.porms.controllers.modal.EditOffenseController;
 import com.rc.porms.data.prefect.offense.OffenseDao;
-import com.rc.porms.data.prefect.offense.impl.OffenseDaoImpl;
+import com.rc.porms.data.prefect.offense.dao.impl.OffenseDaoImpl;
```

Comment:
what is this for?

Reply:
my apology  i forgot to include in the description that I modified the package for the offense dao impl . Because  while refactoring the violation functionality i recognize that the packaging for offense impl  should be  import com.rc.porms.data.prefect.offense.dao.impl.OffenseDaoImpl; but in  my case my previous commit of the offense packaging was insufficient. thats why i had to modify to correct the packaging po.